networking/handshake_handlers.go

```diff
@@ -568,7 +568,7 @@ func (nm *Handshake) checkCompatibility(peerVersion string) bool {
 		// peers major version is too low
 		return false
 	case major > MajorVersion:
-		nm.log.Warn("Peer attempted to connect with a higher major version, this client may need to be updated")
+		nm.log.Debug("peer attempted to connect with a higher major version, this client may need to be updated")
```

Comment:
This and the other two version warnings below should be at least `Info messages`, as we want to prompt the user to update.